exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,4 +1,7 @@
-// Write your function here
+function createGreeting(){
```

Comment:
the parameter is missing inside of the function. After, the **return** will become a string taking that parameter.